SonarQube.MSBuild.Tasks/WriteProjectInfoFile.cs

```diff
@@ -36,6 +38,31 @@ namespace SonarQube.MSBuild.Tasks
     /// that are created.</remarks>
     public class WriteProjectInfoFile : Task
     {
+        #region Fields
+
+        private readonly IEncodingProvider _encodingProvider;
```

Comment:
I probably did this but here people don't use the `_` syntax for fields so I would recommend to get rid of it.